frame/election-provider-multi-phase/src/signed.rs

```diff
@@ -713,9 +726,17 @@ mod tests {
 			assert_eq!(
 				multi_phase_events(),
 				vec![
-					Event::SignedPhaseStarted { round: 1 },
-					Event::SolutionStored { compute: ElectionCompute::Signed, prev_ejected: false },
-					Event::SolutionStored { compute: ElectionCompute::Signed, prev_ejected: false },
+					Event::PhaseTransition { from: Phase::Off, to: Phase::Signed, round: 1 },
+					Event::SolutionStored {
+						compute: ElectionCompute::Signed,
+						origin: RawOrigin::Signed(99),
```

Comment:
I feel a bit icky about putting the origin in the event, how about changing `ElectionCompute::signed(AccountId)`? or `from: Option<accountid>`?